camera_system/game_info.py

```diff
@@ -0,0 +1,200 @@
+"""ゲームエリア情報モジュール.
+
+ゲームエリア情報を保持するクラスを定義している
+@author: kodama0720
+"""
+
+from typing import List, Tuple, Dict
+from robot import Robot, Direction
+from color_changer import Color
+
+
+class GameInfo:
```

Comment:
ファイル名，クラス名，インスタンス名とか諸々，GameAreaInfoとかgame_area_infoに統一しよう
```suggestion
class GameAreaInfo:
```